CHANGELOG.md

```diff
@@ -52,11 +52,14 @@ bootstrap:
                 jobProfileName: Digital helpdesk and Operations User
 ```
 
+## [2.33.0]
```

Comment:
please add the link to the tag in the bottom of the file

Reply:
woops yeah missed it. done!